.github/workflows/run-performance.yaml

```diff
@@ -23,26 +23,113 @@ on:
         required: true
         default: public.ecr.aws/r5b3e0r5/3box/keramik-runner:latest
 
-env:
-  CARGO_TERM_COLOR: always
-  SCCACHE_GHA_ENABLED: "true"
-  RUSTC_WRAPPER: "sccache"
-
 jobs:
-  template-network:
-    name: Template Network
+  run-performance-test:
+    name: Run Performance Test ${{ github.run_id }}
     runs-on: ubuntu-latest
+    environment: cas-scaling-2024
     steps:
       - name: Checkout
         uses: actions/checkout@v3
-      - name: Install jq
-        run: sudo apt-get install jq
+
+      - name: Install yq
+        run: |
+          # sudo apt-get update
+          # sudo apt-get install -y software-properties-common
+          # sudo add-apt-repository -y ppa:rmescandon/yq
+          # sudo apt-get update
+          # sudo apt-get install -y yq
+          curl -L https://github.com/mikefarah/yq/releases/download/v4.40.7/yq_linux_amd64 -o yq
+          chmod +x yq
+          sudo mv ./yq /usr/local/bin/yq
+
       - name: Template network
         run: |
           set -exo pipefail
           export RUST_CERAMIC_IMAGE=${{ inputs.rust_ceramic_image }}
           export SCENARIO_IMAGE=${{ inputs.scenario_image }}
           export JS_CERAMIC_IMAGE=${{ inputs.js_ceramic_image }}
           export TEST_NAME=${{ inputs.test_name }}
-          ls -l tests/performance/$TEST_NAME.yaml
+          ls -l performance/$TEST_NAME
+          export THIS_TEST=${{ inputs.test_name }}-${{ github.run_id }}
+          mkdir -p performance/$THIS_TEST
+          # set the network name yaml key to the test names
+          yq e '.metadata.name = env(THIS_TEST)' performance/$TEST_NAME/network.yaml \
+            > performance/$THIS_TEST/network.yaml
+          yq e '.spec.ceramic[0].image = env(JS_CERAMIC_IMAGE)' -i performance/$THIS_TEST/network.yaml
+          yq e '.spec.ceramic[0].ipfs.rust.image = env(RUST_CERAMIC_IMAGE)' -i performance/$THIS_TEST/network.yaml
+          cat performance/$THIS_TEST/network.yaml
```

Comment:
If not too much work, would you mind moving these lines to the Makefile and invoke via `make` command? If you didn't already know there's no love lost between me and GitHub Actions lol, so the more we use scripts that we can invoke anywhere (including locally), the better IMO.

If it's just a couple of lines, I think it's fine to leave them in the workflow file, but once bash commands start getting a little more voluminous, scripts feel cleaner and more future-safe.